components/form.rst

```diff
@@ -533,19 +533,6 @@ by :method:`Symfony\\Component\\Form\\Form::handleRequest` to determine whether
 
 .. configuration-block::
 
-    .. code-block:: php-standalone
```

Comment:
Was this example removed intentionally?

Reply:
The diff looks weird, maybe use Split View, but from what I can see, it still there 🤔

Reply:
Yep, Oskar's right! Can't tell why the diff looks like this but I just moved the code snippet 🙂